packages/cli/src/services/import.service.ts

```diff
@@ -72,6 +74,14 @@ export class ImportService {
 			if (hasInvalidCreds) await this.replaceInvalidCreds(workflow);
 		}
 
+		// Remove workflows from ActiveWorkflowManager BEFORE transaction to prevent orphaned trigger listeners
+		// This must be done outside the transaction to avoid inconsistent state on rollback
+		for (const workflow of workflows) {
+			if (workflow.id) {
+				await this.activeWorkflowManager.remove(workflow.id);
```

Comment:
Would it make sense to do it in parallel (await Promise.all) ?

Reply:
Sequential execution is intentional here because:
- `activeWorkflowManager.remove()` performs stateful operations (webhook cleanup, queue management)
- The performance gain would be negligible for cleanup operations
- Sequential is safer and more predictable